app-misc/calamares-config-livecd/calamares-config-livecd-3.3.14.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="8"
+
+MY_PN="calamares"
+MY_P="${MY_PN}-${PV}"
+
+inherit desktop
+
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
+DESCRIPTION="Gentoo Linux ${MY_PN} installer config"
+SRC_URI="mirror://gentoo/gentoo-artwork-livecd-2007.0.tar.bz2
+		mirror://gentoo/gentoo-artwork-0.2.tar.bz2"
+
+S="${WORKDIR}"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="app-admin/calamares"
+
+src_install() {
+	insinto "/etc/${MY_PN}"
+	doins -r "${FILESDIR}/modules/"
+	doins -r "${FILESDIR}/settings.conf"
+
+	insinto /usr/lib64/calamares/modules/
```

Comment:
```suggestion
	insinto /usr/$(get_libdir)/calamares/modules/
```

Reply:
> 

this was definitely a reflex. apologies.